### DISCUSSION TITLE
docs/web/router-registering-strict-router-example

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#132] [General]
# docs/web/router-registering-strict-router-example

使用GoFrame框架设置和运行一个简单的Hello服务示例，重点展示了配置文件的使用、路由定义和SwaggerUI页面的自动生成。此外，还探讨了接口文档生成、返回中间件处理以及如何定义规范化接口路径等问题，为开发者提供了一种便捷的方法来生成和测试API。

https://goframe.org/docs/web/router-registering-strict-router-example

**Comment by doodle971002** [comment]
可否提供一下各类型请求(/GET/PUT/POST/DELETE)以及各类型参数位置(header/path/query/cookie)的示例？

**Reply by HelloAllenZhu** [reply]
这个config.yaml的路径是hack/config.yaml?

**Comment by xiaoyi510** [comment]
如果是涉及到  mime:"text/html" 返回纯文本，又应该如何设计 `XXXRes`呢，又该如何返回数据
